docs/toml.md

```diff
@@ -155,11 +155,16 @@ Supported filters:
 ## Access log definition
 
 The standard access log uses the textual Common Log Format (CLF), extended with additional fields.
-Alternatively logs can be written in JSON. 
-Using the default CLF option is simple, e.g.
+Alternatively logs can be written in JSON.
+By default logs will be written to stdout and use the CLF format.
```

Comment:
I rephrased the configuration a bit to make it hopefully more clear and easier to read. Also I added and adjusted documentation in the `traefik.sample.toml` file. Good catch, didn't think about this one..